neon/View/Schedule/Sheets/SchedulePickerView.swift

```diff
@@ -39,7 +39,12 @@ struct SchedulePickerView: View {
             CardsListView {
                 ForEach(viewModel.todaysHourBlocks.filter { $0.hourBlock.hour >= (viewModel.currentDate.isToday ?  viewModel.currentHour : UtilGateway.shared.dayStartHour()) }) { hourBlockViewModel in
                     if hourBlockViewModel.getTitle() != AppStrings.Schedule.HourBlock.empty {
-                        CompactHourBlockView(viewModel: hourBlockViewModel)
+                      CompactHourBlockView(viewModel: hourBlockViewModel)
+                        .contextMenu(ContextMenu(menuItems: {
```

Comment:
You can actually use a short form of this block:
```suggestion
                        .contextMenu {
```